examples/bridge-app/linux/main.cpp

```diff
@@ -71,6 +74,10 @@ EndpointId gFirstDynamicEndpointId;
 Device * gDevices[CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT];
 std::vector<Room *> gRooms;
 std::vector<Action *> gActions;
+#if CHIP_DEVICE_CONFIG_ENABLE_WPA
```

Comment:
This should be conditioned on `CHIP_DEVICE_CONFIG_ENABLE_WIFI`, and should probably have cases for `CHIP_DEVICE_LAYER_TARGET_LINUX` (for Linux) and `CHIP_DEVICE_LAYER_TARGET_DARWIN` (for Darwin, using `NetworkCommissioning::DarwinWiFiDriver`).